packages/react-pdf-toolkit/src/utils/to-title-case.ts

```diff
@@ -0,0 +1,5 @@
+export function toTitleCase(s: string) {
```

Comment:
Let's avoid variable names under 3 characters. 🙏🏼 
```suggestion
export function toTitleCase(str: string) {
```
```suggestion
export function toTitleCase(string_: string) {
```

Reply:
P2